src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorModelOutputUnitTest.java

```diff
@@ -57,7 +57,7 @@ public void testVQSRModelOutput() {
         GATKReport report = vqsr.writeModelReport(goodModel, badModel, annotationList);
         //this generates input data for testVQSRModelInput
         try {
-            PrintStream modelReporter = new PrintStream(publicTestDir+"/"+this.modelReportName);
+            PrintStream modelReporter = new PrintStream(modelReportFile);
```

Comment:
This should become a try with resources, so declare it as such:
`try (PrintStream modelReporter = new PrintStream(modelReportFile);) {`
Which ensures that the stream will be printed

Reply:
Done.